front/src/routes/chat/style.css

```
@@ -178,4 +178,15 @@
 
 .msg_txt_wrap {
   white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)